docs/source/markdown/podman-run.1.md

```diff
@@ -901,17 +901,19 @@ Ulimit options. You can use **host** to copy the current configuration from the
 
 Sets the username or UID used and optionally the groupname or GID for the specified command.
 
-Without this argument the command will be run as root in the container.
+Without this argument the command will be run as root in the container, unless a USER has been
+set in the Containerfile.  If the USER was set in the Containerfile, it will be used to process
+commands within the container.
 
 **--userns**=**auto**|**host**|**keep-id**|**container:**_id_|**ns:**_namespace_
 
-Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value means user namespaces are disabled.
+Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value ("") means user namespaces are disabled.
```

Comment:
the last sentence is not correct, `An empty value ("") means user namespaces are disabled unless an explicit mapping is set with --uidmapping and --gidmapping`

Reply:
Thanks @giuseppe .  I've added ` unless an explicit mapping is set with --uidmapping and --gidmapping` to the end of that sentence as suggested.